Flutter/datagrid/sorting.md

```diff
@@ -473,6 +473,89 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized the sort icon color](images/sorting/flutter-datagrid-customized-sorticon-color.png)
 
+## Set custom sorting icon
+
+`SfDataGrid` allows to change the sort icon through using `SfDataGridThemeData.sortIcon` property. The DataGrid should be wrapped inside the `SfDataGridTheme`. 
```

Comment:
`SfDataGrid` allows you to change the sort icon by using the `SfDataGridThemeData.sortIcon` property.